### DISCUSSION TITLE
Sudent Developer Pack Benefits still not active

### DISCUSSION THREAD

**Discussion by blurryface027** [community/community#138774] [GitHub Education]
### Hi everyone,

Hi everyone, my GitHub Student Developer Pack request was approved, and my account shows as verified. However, the benefits from the pack are still not active or not available. Any advice on how to resolve this would be appreciated. 

Download the best video editor for daily use [CapCut Apk Download](https://capcutapk.co.in)

**Comment by davevad93** [comment]
Hi @blurryface027 ,

Don't worry, you just have to wait, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @blurryface027 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Omkar-bhutale** [reply]
I have been applied multiple time with my college id card but it rejecting always
